external-builds/pytorch/skip_tests/README.md

```diff
@@ -0,0 +1,90 @@
+# Skipping pytorch tests
+
+## Introduction
+
+This tooling allows to narrow down pytorch tests by skipping explicitely tests.
+Either in general, or additioanlly filtered by `amdgpu_family` and/or `pytorch_version`.
+
+By default, we are trying to follow the recommended [PyTorch test suite](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/install/3rd-party/pytorch-install.html#testing-the-pytorch-installation).
+Below you find the extract of it (1st Oct, 2025).
+
+```
+PYTORCH_TEST_WITH_ROCM=1 python3 test/run_test.py --verbose \
+--include test_nn test_torch test_cuda test_ops \
+test_unary_ufuncs test_binary_ufuncs test_autograd
+```
+
+Note:
+1st Oct 2025: Missing is `test/test_ops.py` and `test/inductor/test_torchinductor.py` due to issues in Triton.
+
+However, the filtering using `PYTORCH_TEST_WITH_ROCM=1` is not fully reflecting our failures. As such, this tooling will provide a more fine-grained filtering mechanism in our control to skip additional tests.
+
+Independent of this tooling, it is _always_ welcome to _get those changes upstream_! :)
+
+## How to run
+
+`../run_linux_pytorch_tests.py` steers the pytest and is used by the CI, while `./create_skip_tests.py` creates the list of tests to be included or excluded.
+
+## Structure
+
+Each module to test has its own `skip_test_<name>.py` file and should have the following structure
+
+```
+skip_tests = {
```

Comment:
nit: If you add language names for fenced code blocks GitHub will me more likely to apply syntax highlighting correctly for the code in them

See https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting

````suggestion
```python
skip_tests = {
````